components/playground/playground.go

```diff
@@ -359,7 +360,36 @@ func (p *Playground) handleScaleIn(w io.Writer, pid int) error {
 	return nil
 }
 
-func (p *Playground) sanitizeConfig(boot instance.Config, cfg *instance.Config) {
+func (p *Playground) absDir(dir string) (string, error) {
+	if dir == "" {
+		return "", nil
+	}
+
+	if !filepath.IsAbs(dir) && !strings.HasPrefix(dir, "~/") {
```

Comment:
The `~` should be expanded by the shell in most cases. It's ok to keep this logic, I'm just noting that for records.

Reply:
`tiup playground --db.config ~/db.toml` will expand but `tiup playground --db.config=~/db.toml` will not.